ui/src/components/Form/SelectField/style.module.scss

```diff
@@ -33,6 +33,8 @@
 }
 
 .select-options {
+  height: 12em;
+  overflow-y: scroll;
```

Comment:
Just fyi, you can also use `overflow-y: auto`.

Reply:
ok